test/new-e2e/tests/agent-metric-logs/log-agent/utils/file_tailing_utils.go

```diff
@@ -115,35 +128,36 @@ func checkLogs(s *LinuxFakeintakeSuite, service, content string, expectLogs bool
 	}, 2*time.Minute, 10*time.Second)
 }
 
-// cleanUp cleans up any existing log files (only useful when running dev mode/local runs).
-func (s *LinuxFakeintakeSuite) cleanUp() {
-	t := s.T()
+// CleanUp cleans up any existing log files (only useful when running dev mode/local runs).
+func CleanUp(ls LogsTestSuite) {
+	t := ls.T()
+	t.Helper()
 	var checkCmd string
 
-	if s.DevMode == true {
-		osType := s.Env().VM.GetOSType()
+	if ls.IsDevMode() {
+		osType := ls.Env().VM.GetOSType()
 
 		switch osType {
 		default: // default is linux
-			s.Env().VM.Execute("sudo rm -f /var/log/hello-world.log")
-			s.Env().VM.Execute("sudo rm -f /var/log/hello-world.log.old")
-			checkCmd = "ls /var/log/hello-world.log /var/log/hello-world.log.old 2>/dev/null || echo 'Files do not exist'"
+			ls.Env().VM.Execute("sudo rm -f /var/log/hello-world.log")
+			ls.Env().VM.Execute("sudo rm -f /var/log/hello-world.log.old")
+			checkCmd = "ls /var/log/helzlo-world.log /var/log/hello-world.log.old 2>/dev/null || echo 'Files do not exist'"
```

Comment:
```suggestion
			checkCmd = "ls /var/log/hello-world.log /var/log/hello-world.log.old 2>/dev/null || echo 'Files do not exist'"
```
Was this intentional?

Reply:
o no, thats a typo! fixing that